src/components/Header/TopNav/LogoLink.js

```diff
@@ -22,40 +18,45 @@ const StyledLink = styled(Link)`
     width: ${remcalc(54)};
   }
 `
+const HomepageLogo = styled.img`
+  width: 49px;
+  height: 36px;
+  margin-top: ${remcalc(6)};
+`
+
+const LogoLink = ({ slug, isServicePage, isSpecialityPage, isHomePage }) => {
+  const renderSvg = isServicePage || isSpecialityPage || isHomePage
+
+  const fillColorInitial = logoColors[slug] || logoColors['default']
 
-const LogoLink = ({ path = '/' }) => {
-  const isServicePage = getIsServicePage(path)
-  const isSpecialityPage = path.includes('speciality')
+  const fillColorHover = isSpecialityPage
+    ? logoColors.specialityHover
+    : logoColors.defaultHover
 
```

Comment:
Same with these 2: fillColorHover and defaultHover:
`const fillColorHover = logoColors[isSpecialityPage ? 'specialityHover' : 'defaultHover']

Reply:
@GBouffard  same thing here